index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <section class="tool-bar">
+        <a class="tool-bar-logo" href="index.html">Alan Luqman</a>
+        <svg class="menu-button" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18
+            1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18
+            6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523
+            12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+        </svg>
+    </section>
+    <section class="heading">
+        <div class="heading-container">
+            <h1 class="primary-text">Hey There. I'm Alan</h1>
+            <p class="support-text">I'm a Software Developer</p>
+            <p class="description">
+                I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                 If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </p>
+            <ul class="social-media-list">
+                <li class="social-media-icon">
+                    <svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M17.9989 7.46114C17.9989 6.32124 17.6329 5.25043 16.9342 4.31779C17.2004 3.41969 17.2669
+                    2.27979 17.2004 0.967185C17.1671 0.414508 16.7345 0 16.2022 0C15.9028 0 13.3741 0.0345423 11.9101
+                    1.38169C10.6458 1.1399 9.31495 1.1399 8.01736 1.38169C6.58667 0.0345423 4.05802 0 3.7253 0C3.19296
+                    0 2.76042 0.414508 2.72715 0.967185C2.62734 2.27979 2.72715 3.41969 2.99333 4.31779C2.29462 5.28497
+                    1.92863 6.35579 1.92863 7.46114C1.92863 9.8791 3.7253 12.0553 6.45358 13.0915C6.35377 13.2988 6.28723
+                    13.5406 6.22068 13.7824C3.32604 13.4715 1.9619 10.7427 1.89536 10.639C1.66246 10.1209 1.06357 9.91364
+                    0.56449 10.19C0.0654144 10.4318 -0.134216 11.0535 0.131958 11.5717C0.198501 11.7444 2.02845 15.4404
+                    6.05432 15.8549V18.9637C6.05432 19.5509 6.48686 20 7.05248 20H12.875C13.4406 20 13.8732 19.5509 13.8732
+                    18.9637V14.8532C13.8732 14.2314 13.7401 13.6442 13.5072 13.1261C16.2022 12.0553 17.9989 9.91364 17.9989
+                    7.46114Z" fill="#DBD8D7"/>
+                    </svg>
```

Comment:
- [ ] Kindly wrap each `SVG` icon on this list in an `a` tag. The idea here is to drive visitors to your website to your social media platforms in case one may want to contact you on any of them for a potential job or networking purposes. Please fix that for all the li items in this section for the social media icons. 👍